### ISSUE TITLE
Cart Page regression: mouse-clickable elements are not keyboard accessible

### ISSUE THREAD

**Issue by developpeurweb** [SAP/spartacus#14872]
**Describe the bug**
- product image is not keyboard focusable, it matters because it's a link.
- product name is keyboard focusable but doesn't have a focus ring.
- both elements are mouse-clickable, so this is a keyboardability regression.

**Tell us the version of Spartacus**
- Library version:  4.3 / develop

**To Reproduce**
Steps to reproduce the behaviour:
1. Add a couple of products to the cart.
2. Access the Cart Page using the keyboard only.
3. Tab to the products you added.
4. Notice the image is not focusable and the product name is missing the visual focus ring.

**Expected behavior**
- both links: image and product name are keyboard accessible and show a focus ring while receiving focus.

**Screenshots**

![2022-01-24_14-07-34](https://user-images.githubusercontent.com/15113219/151055622-0e62efe4-4e5e-406d-9ead-9c6bb79b556b.jpg)

**Comment by KateChuen** [comment]
Tested on develop branch of spartacus using installation script
More extensive testing (on other screenreaders and devices) will be done with RC.

Confirmed that mini-cart is keyboard accessible and goes to cart page.
On cart page, confirmed that both product image and product title are keyboard accessible, have the focus ring when in focus, and land on the pdp page when accessed. 

![minicart](https://user-images.githubusercontent.com/45409515/155799346-1faf2c6f-c44b-4723-8d1e-8c7f0ad35445.png)

![productImage](https://user-images.githubusercontent.com/45409515/155799336-f2bf0203-8b47-4dc2-b7d1-4b39c9604cd3.png)

![productTitle](https://user-images.githubusercontent.com/45409515/155799340-112ac420-1182-4234-9a75-bf1d9d7cc115.png)
